lib/quandl/connection.rb

```diff
@@ -5,7 +5,7 @@ def self.request(http_verb, url, options = {}, &block)
       headers = options.delete(:headers) || {}
       accept_value = 'application/json'
       accept_value += ", application/vnd.quandl+json;version=#{ApiConfig.api_version}" if ApiConfig.api_version
-      headers = { accept: accept_value }.merge(headers)
+      headers = { accept: accept_value, request_source: 'ruby', request_source_version: Quandl::VERSION }.merge(headers)
```

Comment:
here is the request RestClient sends up:
```
=> #<RestClient::Request:0x007fac10370708
 @args=
  {:url=>"https://www.quandl.com/api/v3/datasets/NSE/OIL/metadata",
   :headers=>{:accept=>"application/json", :request_source=>"ruby", :request_source_version=>"1.0.1"},
   :method=>:get},
 @block_response=nil,
 @cookies={},
 @headers={:accept=>"application/json", :request_source=>"ruby", :request_source_version=>"1.0.1"},
 @max_redirects=10,
 @method=:get,
 @password=nil,
 @payload=nil,
 @processed_headers=
  {"Accept"=>"application/json",
   "Accept-Encoding"=>"gzip, deflate",
   "Request-Source"=>"ruby",
   "Request-Source-Version"=>"1.0.1"},
```

Also I believe "X-Api-Version" header is set the same way with underscores in the symbol name:
https://github.com/quandl/quandl-ruby/blob/master/lib/quandl/connection.rb#L9